src/librustc_typeck/check/writeback.rs

```diff
@@ -355,22 +354,6 @@ impl<'cx, 'gcx, 'tcx> WritebackCx<'cx, 'gcx, 'tcx> {
         }
     }
 
-    fn visit_cast_types(&mut self) {
```

Comment:
This function is not modifying anything. So removed it.

Reply:
Maybe it needs to call `set_coercion_cast?`